services/web/server/tests/unit/with_dbs/06/test_projects_02.py

```diff
@@ -908,16 +909,53 @@ async def test_tags_to_studies(
     await assert_status(resp, web.HTTPNoContent)
 
 
+@pytest.fixture
+def client_on_running_server_factory(client: TestClient, loop) -> Iterator[Callable]:
+    # Creates clients connected to the same server as the reference client
+    #
+    # Implemented as aihttp_client but creates a client using a running server,
+    #  i.e. avoid client.start_server
+
+    assert isinstance(client.server, TestServer)
+
+    clients = []
+
+    def go():
```

Comment:
maybe a more meaningful name would help here

Reply:
haha ... this is a modification of ``aiohttp.pytest_plugin`` 
It is a standard used in all libraries under https://github.com/aio-libs. You find it everywhere.
``go`` to initialize something and ``finalize`` to tear-down

Reply:
I need context to comprehend the context. I see! 🤣